### PR TITLE
Update how-to-format-your-csv-file-for-data-content-migration.md

### PR DESCRIPTION
**Pull request by modery**
Some smaller fixes and some clarifications on what needs to go into the CSV file

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/how-to-format-your-csv-file-for-data-content-migration.md [hunk 1]
```diff
@@ -172,16 +172,16 @@ The format is similar to regular bulk upload files, except there are two additio
 |Target Web | *Required*. Enter the SharePoint site URL where the files are migrating to. |
 |Target DocLib | *Required*. Enter the name of the document library with the SharePoint site where the files are migrating to. |
 |Target SubFolder | *Optional*. Enter the name of the subfolder in the document library. If this column is left empty, the files will be moved to the root level. |
-|RegisterAsHubSite|Indicate if you want the destination site be be a hub site. Enter **YES** or **NO**. If you say **YES**, do not put a value in the *AssociateWithHubURL* column.  Nested hub sites are not supported.|
-|AssociateWithHubURL|Enter the URL of an existing hub site you want to associate your destination site with. You cannot have a **YES** value in the *RegisterAsHubSite* column if you have a value here.  Nested hub sites are not supported.|
+|RegisterAsHubSite|Indicate if you want the destination site be be a hub site. Enter **YES** or **NO**. If you say **YES**, do not put a value in the *AssociateWithHubURL* column.Nested hub sites are not supported.|
```
Comment (JohanFreelancer9) [inline migration/how-to-format-your-csv-file-for-data-content-migration.md:175]:
```suggestion
|RegisterAsHubSite|Indicate if you want the destination site be be a hub site. Enter **YES** or **NO**. If you say **YES**, do not put a value in the *AssociateWithHubURL* column.Nested hub sites are not supported.|
```
```suggestion
|RegisterAsHubSite|Indicate if you want the destination site to be a hub site. Enter **YES** or **NO**. If you say **YES**, do not put a value in the *AssociateWithHubURL* column. Nested hub sites are not supported.|
```

Comment (JohanFreelancer9) [inline migration/how-to-format-your-csv-file-for-data-content-migration.md:175]:
```suggestion
|RegisterAsHubSite|Indicate if you want the destination site to be a hub site. Enter **YES** or **NO**. If you say **YES**, do not put a value in the *AssociateWithHubURL* column. Nested hub sites are not supported.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @modery, kindly commit the suggested changes. Many thanks.

**Comment by modery** [comment]
Suggestions committed, all ok for me

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**